.github/workflows/request-reviewer.yml

```
@@ -15,7 +15,7 @@ on:
 
 jobs:
   request-review:
-    if: (github.event.action == 'created' && github.event.issue.state == 'open' && github.event.issue.draft == false && github.event.comment.user.login == github.event.issue.user.login) || (github.event.action != 'created' && github.event.pull_request.state == 'open' && github.event.pull_request.draft == false )
+    if: (github.event.issue.state == 'open' && github.event.issue.draft == false) && (github.event.action != 'created' || github.event.comment.user.login == github.event.issue.user.login)
```

wait no I was wrong - it has to account for the difference between events where the issue is called `issue` and where it's called `pull_request`

```suggestion
    if: (github.event.action == 'created' && github.event.issue.state == 'open' && github.event.issue.draft == false && github.event.comment.user.login == github.event.issue.user.login) || (github.event.action != 'created' && github.event.pull_request.state == 'open' && github.event.pull_request.draft == false )
```

---

mm yeah that makes sense. It's why I had written it that way originally. I assumed you knew that the issue object was always present or similar.

---

nope! I just forgot 😂  it's way too easy to miss!